android/src/main/java/com/horcrux/svg/PathParser.java

```diff
@@ -62,7 +62,7 @@ static Path parse(String d) {
 
       if (!has_prev_cmd && first_char != 'M' && first_char != 'm') {
         // The first segment must be a MoveTo.
-        throw new Error(String.format("Unexpected character '%c' (i=%d, s=%s)", first_char, i, s));
+        throw new IllegalArgumentException(String.format("Unexpected character '%c' (i=%d, s=%s)", first_char, i, s));
```

Comment:
Why changing it from `Error` to `IllegalArgumentException` fixes the issue? Do we catch it somewhere and handle properly then?

Reply:
That fix displays the error and the reason why it happened without crashing the application.